tests/test_util.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from numpy.testing import assert_allclose
+import pytest
+import sfs
+
+test_data = [
```

Comment:
Since we might add more tests for 'util' in the future its a good idea to use a more specific name for the array.